### PR TITLE
Additional Info and Individual API Changes

### PR DESCRIPTION
**Pull request by lalithkota**
- Added Additional Info field which is a postgres JSON Field.
- Added Individual related Rest APIs.
- Reorganized Rest APIs with reuse code, with mixins.

### CODE REVIEW

File: g2p_registry_addl_info/models/json_field.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import json
```
Comment (shibu-narayanan) [inline g2p_registry_addl_info/models/json_field.py]: Can we move json_field outside or registry. So that it can be in use independent of registry

Comment (lalithkota) [reply g2p_registry_addl_info/models/json_field.py]: @shibu-narayanan ok sure. What's the best place to put this ?

File: g2p_registry_addl_info/static/src/js/g2p_additional_info.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                if (typeof valuesJsonOrig[key] === "object") {
+                    sections[key] = this.flattenJson(valuesJsonOrig[key]);
+                } else {
+                    valuesJson[key] = valuesJsonOrig[key];
+                }
+            }
+            if (Object.keys(sections).length === 0) {
+                sections = false;
+            }
+            console.debug(sections);
```
Comment (shibu-narayanan) [inline g2p_registry_addl_info/static/src/js/g2p_additional_info.js:38]: console writes can be avoided.

**Review by shibu-narayanan — APPROVED** [review]

**Review by pjoshi751 — APPROVED** [review]
Ok to merge, but needs sufficient testing.

**Pull request merged**